release-team/role-handbooks/docs/Release-Timeline.md

```diff
@@ -325,17 +334,42 @@ git commit -m "Merge master into dev-[future release] to keep in sync"
 git push origin merged-master-dev-[future release]
 ```
 
-Submit a PR against upstream `dev-[future release]` from your fork's branch `merged-master-dev-[future release]`. e.g., [merge master into future release](https://github.com/kubernetes/website/pull/16225).
-
 You may need to fix conflicts manually. If somebody has improved a page on `master`, and at the same time it has been updated in the dev branch for the next release, we may need to figure out how to make those changes work together. If something comes up which isn't obvious, you can always abort the merge and reach out to SIG Docs for help.
 
 When you have completed resolving the differences manually, run `git merge --continue` to complete the merge. Then carry on from step 5.
 
-⚠️  **Ensure the PR is passing tests on GitHub**.
+> Rebase Method
```

Comment:
Hi, @eagleusb 👋 Is there a reasoning behind adding the `rebase` method to the documentation? `Merge` method is preferred because it can be later reviewed by folks if needed(as it preserves the history). 
@jimangel @sftim @kbhawkey - I am not sure where `k/website` stands with rebasing a branch into another. Can you provide some insight into this, please? 

/hold

Reply:
Any input on preference on merging vs rebasing would really help! Maybe we can add the reasoning to docs as well.

I also thought about why we merge instead of rebasing and clarifying this will help future shadows who might have the same questions.

Reply:
I want to say it has to do with preserving the history of commits. I am not totally sure,so will wait to hear from other folks.

Reply:
IMO [k/website](https://github.com/kubernetes/website) can and should use merge by default, with squash-then-merge as a fallback to help contributors less familiar with Git.

Reply:
Asking the docs release team to know how to do a merge rebase (`git rebase --rebase-merges`) doesn't seem like a reasonable expectation, either.

Reply:
Thanks for your review and help @savitharaghunathan @annajung @sftim 

The reasoning was just to have a clearer history in a dev branch while tracking master changes, thus the need for rebasing (graph readability).
I thought that dev branches were eventually merged and deleted but it seems that there are long lived ones.

Thanks for the explanation and time, I'm closing that PR then.